Shared/Extensions/UI Extensions/UIColor+Extensions.swift

```diff
@@ -11,6 +11,24 @@ import UIKit
 
 extension UIColor {
 
+    var hue: CGFloat {
```

Comment:
I'm using these to group similar colors together on the gradient view. The idea is to make the gradient smoother.